services/QuillLMS/config/puma.rb

```diff
@@ -12,8 +12,7 @@
 on_worker_boot do
   # worker specific setup
   ActiveSupport.on_load(:active_record) do
-    config = ActiveRecord::Base.configurations[Rails.env] ||
-             Rails.application.config.database_configuration[Rails.env]
+    config = Rails.application.config.database_configuration[Rails.env]['primary']
```

Comment:
I don't see this mentioned in the PR note. Is it supposed to be here?

Reply:
Nope! Thank you for pointing that out.